spec/system/claims/support/create_a_claim_window_spec.rb

```diff
@@ -22,19 +22,19 @@
     then_i_see_a_form_error("Enter a window opening date in the future")
     then_i_see_a_form_error("Enter a window closing date in the future")
```

Comment:
Could we please cover these validation rules in a unit test as well/instead? I think that would be a clearer way to cover the validation rule without needing to drive the UI as a user.